src/resolve/resolved-dns-query.c

```diff
@@ -585,6 +586,17 @@ void dns_query_complete(DnsQuery *q, DnsTransactionState state) {
 
         q->state = state;
 
+        if (state == DNS_TRANSACTION_SUCCESS && set_size(q->manager->varlink_subscription) > 0) {
+                DnsQuestion *question = q->request_packet ? q->request_packet->question :
+                                                            NULL;
+                const char *query_name = question ? dns_question_first_name(question) : q->request_name;
+                if (query_name != NULL) {
+                        int r = send_dns_notification(q->manager, q->answer, query_name);
+                        if (r < 0)
+                                log_error_errno(r, "Failed to send varlink notification, ignoring: %m");
```

Comment:
If the error is ignored, then please downgrade the error level. I think `log_debug_errno()` is enough.

Reply:
Hmm, if `send_dns_notification()` is failed, we have two log message about the failure.
Please downgrade all log messages in `send_dns_notification()` to debug level.